packages/config-cli/src/cli/screenshot/screenshot.ts

```diff
@@ -39,104 +46,100 @@ export class CommandScreenShot extends CommandLineAction {
-      searchParam.set('p', test.tileMatrix);
-      searchParam.set('i', tileSetId);
-      if (styleId) searchParam.set('s', styleId);
+    const searchParam = new URLSearchParams();
+    searchParam.set('p', test.tileMatrix);
+    searchParam.set('i', test.tileSet);
+    if (test.style) searchParam.set('s', test.style);
 
-      const loc = `@${test.location.lat},${test.location.lng},z${test.location.z}`;
-      const fileName = '.artifacts/visual-snapshots/' + host + '_' + test.name + '.png';
+    const loc = `@${test.location.lat},${test.location.lng},z${test.location.z}`;
+    const fileName = '.artifacts/visual-snapshots/' + host + '_' + test.name + '.png';
 
-      await mkdir(`.artifacts/visual-snapshots/`, { recursive: true });
+    await mkdir(`.artifacts/visual-snapshots/`, { recursive: true });
 
-      const url = `https://${host}/?${searchParam.toString()}&debug=true&debug.screenshot=true#${loc}`;
+    let url = `${host}/?${searchParam.toString()}&debug=true&debug.screenshot=true#${loc}`;
+    if (!url.startsWith('http')) url = `https"//${url}`;
 
-      logger.info({ url, expected: fileName }, 'Page:Load');
+    logger.info({ url, expected: fileName }, 'Page:Load');
 
-      await page.goto(url);
+    await page.goto(url);
 
-      try {
-        await page.waitForSelector('div#map-loaded', { state: 'attached' });
-        await page.waitForTimeout(1000);
-        await page.waitForLoadState('networkidle');
-        await page.screenshot({ path: fileName });
-      } catch (e) {
-        await page.screenshot({ path: fileName });
-        throw e;
-      }
-      logger.info({ url, expected: fileName }, 'Page:Load:Done');
-      await page.close();
+    try {
+      await page.waitForSelector('div#map-loaded', { state: 'attached' });
+      await page.waitForTimeout(1000);
+      await page.waitForLoadState('networkidle');
+      await page.screenshot({ path: fileName });
+    } catch (e) {
+      await page.screenshot({ path: fileName });
+      throw e;
     }
+    logger.info({ url, expected: fileName }, 'Page:Load:Done');
+    await page.close();
   }
+}
 
-  async getTileSetId(tileSetId: string, tag: string): Promise<string> {
-    if (tag === 'production') return tileSetId;
-
-    const tileSetTagId = `${tileSetId}@${tag}`;
-    const dbId = Config.TileSet.id(tileSetTagId);
-    const tileSet = await Config.TileSet.get(dbId);
-
-    if (tileSet) return tileSetTagId;
-    return tileSetId;
-  }
-
-  async getStyleId(styleId: string | undefined, tag: string): Promise<string> {
-    if (styleId == null) return '';
-    if (tag === 'production') return styleId ?? '';
-
-    const styleIdTagId = `${styleId}@${tag}`;
-    const dbId = Config.Style.id(styleIdTagId);
-    const style = await Config.Style.get(dbId);
-    if (style) return styleIdTagId;
-    return styleId;
-  }
+async function startServer(host: string, port: number, config: string, logger: LogType): Promise<FastifyInstance> {
```

Comment:
why is host passed in just to log it?

could generate host inside the function or log outside the function